src/main/java/com/stack/project/Impl/stackImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.stack.project.Impl;
+
+import java.util.Vector;
+
+import com.stack.project.stack;
+
+public class stackImpl <E> implements stack<E>  {
+
+    private Vector<E> vec;
+
+    public stackImpl() {
+        vec = new Vector<>(); 
+    }
+    
+    @Override
+    public boolean empty() {
```

Comment:
change to isEmpty, we want thar a function that returns a boolean will be called with a name that will hint on its return value. so isSomething return is true or false.